analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/CalculationsShouldNotOverflow.CSharp9.cs

```diff
@@ -3,7 +3,10 @@
 using System.Linq;
 
 int i = 2147483600;
-i += 100; // FN, Top level statements are not supported
+i += 100; // FN
```

Comment:
not sure why this FN did not turn to FP  - I added another case below and it seems to be normally detected.

Reply:
We don't support compound assignment. So all the UTs here and below are in fact wrong, including their comments.

Please update them all to 
```
        int i = 2147483600;
        _ = i + 100;
```
and that should make those false FNs into Noncompliant